test/validateSchema.js

```diff
@@ -0,0 +1,75 @@
+'use strict';
+
+const path = require('path');
+const metatests = require('metatests');
+const metaschema = require('..');
+
+const schemaDir = name => path.join(__dirname, 'schemas', name);
+
+const test = config => metatests.test(config.name, test => {
+  const dir = schemaDir(config.schemaName);
+  metaschema.fs.loadAndCreate(dir, null, error => {
+    if (!config.error) {
+      test.error(error, 'must not return error');
+    } else {
+      test.assert(error, 'must return errors');
+      test.equal(error.errors.length, 1, 'must only return one error');
+      const type = config.error.type;
+      const source = config.error.source;
+      const property = config.error.property;
+      const [actual] = error.errors;
+      const valid = (
+        actual.type === type &&
+        actual.source === source &&
+        actual.property === property
+      );
+      test.assert(valid, 'must return appropriate error');
```

Comment:
Why not have 3 asserts instead of manually creating `valid`?
i.e.
```js
test.strictSame(actual.type, config.error.type);
test.strictSame(actual.source, config.error.source);
test.strictSame(actual.property, config.error.property);
```